ansible/config_sonic_basedon_testbed.yml

```diff
@@ -515,6 +515,14 @@
         become: true
         shell: config load_minigraph -y
 
+      - name: config static route for trex traffic passthrough
+        become: true
+        command: "{{ item }}"
```

Comment:
Just performed config reload on switch. The switch may be unreachable yet.
It would be better to run these commands after the next task `Wait for switch to become reachable again`.

Reply:
Thanks Xin, will change it!